src/decaf/points.rs

```diff
@@ -492,6 +495,7 @@ impl TryFrom<Box<[u8]>> for CompressedDecaf {
     }
 }
 
+#[cfg(any(feature = "alloc", feature = "std"))]
```

Comment:
Why is this necessary? AFAIK, no allocations occur in this code block?

Reply:
compiler can't find String type here

Reply:
no allocations, but also no `extern alloc` and the compiler cannot find `String`